src/Commands/UninstallHooks.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Feek\LaravelGitHooks\Commands;
+
+use Feek\LaravelGitHooks\Traits\GitHookDelimiter;
+use Illuminate\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class UninstallHooks extends BaseCommand
+{
+
+    use GitHookDelimiter;
+
+    /**
+     * @var Finder
+     */
+    private $finder;
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * InstallHooks constructor.
+     *
+     * @param Finder $finder
+     * @param Filesystem $filesystem
+     */
+    public function __construct(Finder $finder, Filesystem $filesystem)
+    {
+        parent::__construct();
+        $this->finder = $finder;
+        $this->filesystem = $filesystem;
+    }
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'hooks:uninstall';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Remove all hooks inside .git/hooks/';
```

Comment:
nitpick, but could you reword this to "Remove all hooks generated by the laravel git hooks package"?

Reply:
No problem, just pushed a change with a new description.